test/acl.test.js

```diff
@@ -5,10 +5,13 @@
 
 'use strict';
 var assert = require('assert');
+var expect = require('./helpers/expect');
 var loopback = require('../index');
 var Scope = loopback.Scope;
 var ACL = loopback.ACL;
 var request = require('supertest');
+var Promise = require('bluebird');
+var supertest = require('supertest-as-promised')(Promise);
```

Comment:
FWIW, the recent versions of supertest provide Promise API natively, supertest-as-promised was deprecated. Let's wait for https://github.com/strongloop/loopback/pull/3196 to be landed.